frontend/src/components/modals/AddTeamMemberModal.tsx

```diff
@@ -68,24 +71,10 @@ export const AddTeamMemberModal: Modal<ModalTypes.ADD_TEAM_MEMBER_MODAL> = ({
       </ModalHeader>
       <ModalContent>
         <div className={classes(css.addMemberContent)}>
-          <div className={classes(css.instructions)}>
-            {openInfo && (
-              <>
-                <b>
-                  <Trans i18nKey="Here’s how to add a team member" />
-                </b>{' '}
-                <Trans i18nKey="Team member addition instructions" />{' '}
-              </>
-            )}
-            <button
-              className={classes(css.linkButton, css.blue)}
-              tabIndex={0}
-              type="button"
-              onClick={() => setOpenInfo((prev) => !prev)}
-            >
-              <Trans i18nKey={openInfo ? 'Hide info' : 'Show info'} />
-            </button>
-          </div>
+          <AddTeamMemberInfo
+            open={openInfo}
+            onChange={(open) => setOpenInfo(open)}
```

Comment:
The `setOpen` could be passed directly
```suggestion
            onChange={setOpenInfo}
```